.travis.yml

```diff
@@ -22,8 +22,8 @@ sudo: false
 #dist:
```

Comment:
And looking at latest build logs, it complains about cmake (absence of recipes). Is that part of the build disabled too?